src/index.html

```diff
@@ -9,7 +9,105 @@
   <link rel="stylesheet" href="./style.css">
 </head>
 <body>
-  <h1>HTML Form</h1>
+  <form method="post" action="https://mate.academy/ru">
+    <fieldset class="form-wrapper">
+      <legend>Personal information</legend>
+      <div class="surname">
+        <label class = "label" >Surname:
```

Comment:
```suggestion
        <label class="label">Surname:
```
remove redundant spaces